web/src/pages/Team/CreateTeam.tsx

```diff
@@ -0,0 +1,315 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import '../EditMission/Mission.scss';
+import '../Settings/Settings.scss';
+import {
+    FormControl,
+    InputLabel,
+    Select,
+    MenuItem,
+    SelectChangeEvent,
+    OutlinedInput,
+    Box,
+    Chip,
+} from '@mui/material';
+import { useNavigate } from 'react-router-dom';
+import Cookies from 'js-cookie';
+import Feedbacks from '../../component/Feedback';
+import TopBar from '../../component/SideBar/TopBar';
+import SideBar from '../../component/SideBar/SideBar';
+import config from '../../config';
+import { Input } from '../EditMission/EditMission';
+
+const ITEM_HEIGHT = 55;
+const ITEM_PADDING_TOP = 8;
+const MenuProps = {
+    PaperProps: {
+        style: {
+            maxHeight: ITEM_HEIGHT * 4.5 + ITEM_PADDING_TOP,
+            width: 250,
+        },
+    },
+};
+
+export default function CreateTeam() {
+    const [Title, setTitle] = useState('');
+    const [Team, setTeam] = useState<string[]>();
+    const [open, setOpen] = useState(false);
+    const [message, setMess] = useState<{ mess: string; color: string }>({
+        mess: '',
+        color: 'success',
+    });
+    const [penList, setListPen] = useState<{ id: number; name: string }[]>([]); // recupère la liste des pentester dispo
+    const navigate = useNavigate();
+    const [personName, setPersonName] = useState<string[]>([]);
+    const [manager, setManager] = useState(''); // reécupère la liste des pen à mettre dans la team
+    const [managerList, setManagerList] = useState<
+        { id: number; name: string }[]
+    >([]);
+
+    const handleChange = (event: SelectChangeEvent) => {
+        const {
+            target: { value },
+        } = event;
+        setPersonName(typeof value === 'string' ? value.split(',') : value);
+        const tab = [];
+        for (let i = 0; i < value.length; i += 1) {
+            const note = penList.filter((elem) => elem.id === Number(value[i]));
+            tab.push(note[0].name);
+        }
+        setTeam(tab);
+    };
+
+    const getManager = async () => {
+        await axios
+            .get(`${config.apiUrl}/manager?page=1`, {
```

Comment:
can you add an issue for pagination implementation, or leave a TODO somewhere ? feels like we will forget about this next month